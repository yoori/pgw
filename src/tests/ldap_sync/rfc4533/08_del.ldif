#
# ARGV: -x -H ${RFC4533_TEST_SERVER} -D "cn=admin,dc=example,dc=com" -w "secret"
# OUT:groupDelete
#
dn: cn=baz,ou=groups,dc=example,dc=com
changetype: delete

