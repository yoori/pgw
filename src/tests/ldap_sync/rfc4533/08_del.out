Delete ou=groups,dc=example,dc=com cn=baz,ou=groups,dc=example,dc=com 
